app/views/layouts/_flash_notices.html.erb

```diff
@@ -13,7 +13,11 @@
         There is a problem
       </h2>
       <div class="govuk-error-summary__body">
-        <p><%= msg %></p>
+        <ul class="govuk-list govuk-error-summary__list">
+          <li>
+            <a href="#6-digit-error-summary"><%= msg %></a>
```

Comment:
I think this `<a>` tag should't be here. I know it is in the design system, but it is specific to the 6 digit error and this layout is shared between a lot of views